src/control/server/ctl_system.go

```diff
@@ -89,7 +89,8 @@ func (svc *ControlService) updateMemberStatus(ctx context.Context) error {
 
 		for _, result := range hResults {
 			if result.State == system.MemberStateUnresponsive ||
-				result.State == system.MemberStateStopped {
+				result.State == system.MemberStateStopped ||
+				result.State == system.MemberStateErrored {
```

Comment:
There's a TODO comment above that I believe has to do with this, which should no longer be a TODO